config/database.yml

```diff
@@ -1,25 +1,49 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
+# PostgreSQL. Versions 8.2 and up are supported.
 #
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem "sqlite3"
+# Install the pg driver:
+#   gem install pg
+# On OS X with Homebrew:
+#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
+# On OS X with MacPorts:
+#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
+# On Windows:
+#   gem install pg
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
 #
+# Configure Using Gemfile
+# gem 'pg'
+#
+# port: 5432
+#
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
+  host: localhost
+  encoding: unicode
+  port: 5432
+  user: postgres
+  password: 1234
+  min_messages: error
+
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: db/development
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *default
```

Comment:
We need to use a different config for the test. Rails should create and delete this database every time you run your tests.
This article should explain how to do that https://www.digitalocean.com/community/tutorials/how-to-set-up-ruby-on-rails-with-postgres. Should not be a huge change.